--- init.rc	2017-01-11 19:06:35.678439300 +0530
+++ init.rc.edited	2017-01-11 19:02:31.000000000 +0530
@@ -13,6 +13,7 @@
 import /init.carrier.rc
 import /init.container.rc
 import /init.rilcommon.rc
+import /init.sickness.rc
 
 on early-init
     # Set init and its forked children's oom_adj.
@@ -719,8 +720,8 @@
     chmod 0444 /sys/devices/system/soc/soc0/soc_pvs
 
     # Assume SMP uses shared cpufreq policy for all CPUs
-    chown system system /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
-    chmod 0660 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
+    chown root root /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
+    chmod 0640 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
 
     # Auto Brightness
     chown system system  /sys/class/backlight/panel/auto_brightness
@@ -1868,3 +1869,15 @@
 on property:service.vpnclientd.enable=0
     stop vpnclientd
 
+# Call Post-init script
+service sick-init /sbin/sick.sh
+    class main
+    user root
+    oneshot
+
+# init.d support enabler
+service sysinit /system/bin/logwrapper /system/xbin/busybox run-parts /system/etc/init.d
+    class late_start
+    user root
+    group shell
+    oneshot
