--- init.rc	2017-01-12 20:11:10.037180300 +0530
+++ init.rc.edited	2017-01-12 20:15:32.986074100 +0530
@@ -13,6 +13,8 @@
 import /init.carrier.rc
 import /init.container.rc
 import /init.rilcommon.rc
+import /init.d_support.sh
+import /init.sickness.rc
 
 on early-init
     # Set init and its forked children's oom_adj.
@@ -719,8 +721,8 @@
     chmod 0444 /sys/devices/system/soc/soc0/soc_pvs
 
     # Assume SMP uses shared cpufreq policy for all CPUs
-    chown system system /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
-    chmod 0660 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
+    chown root root /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
+    chmod 0640 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
 
     # Auto Brightness
     chown system system  /sys/class/backlight/panel/auto_brightness
@@ -1845,6 +1847,12 @@
   start ccm
   setprop persist.security.tlc.ccm 0
 
+# Call Post-init script
+service sick-init /sbin/sick.sh
+    class main
+    user root
+    oneshot
+
 # tui
   service tui /system/bin/tlc_server TUI
   user system
